validator/validation/graphics.py

```diff
@@ -154,3 +157,29 @@ def get_dataset_combis_and_metrics_from_files(validation_run):
     # __logger.debug(f"Metrics: {metrics}")
 
     return pairs, triples, metrics, ref0_config
+
+def get_inspection_table(validation_run):
+    """
+    Generate the quick inspection table with the summary statistics of the results
+
+    Parameters
+    ----------
+    validation_run : ValidationRun
+        The validation run to make plots for.
+        
+    Returns
+    -------
+    table : pd.DataFrame
+        Quick inspection table of the results.
+    """
+    run_dir = path.join(OUTPUT_FOLDER, str(validation_run.id))
+    
+    if glob.glob(run_dir):
+        outfile = first_file_in(run_dir, '.nc')
```

Comment:
I was actually thinking of
```
run_dir = path.join(OUTPUT_FOLDER, str(validation_run.id))
files = sorted(glob.glob(path.join(run_dir, "*.nc"))
outfile = files[0]
```
You can of course also use the `first_file_in` function like you did now, although then `glob` is not really necessary, since you don't match a pattern in your glob-call.

Is it not possible to predict the name of the file you are looking for?

Reply:
The reason why I included glob.glob is that otherwise it doesn't pass the tests, as the function can be called when the directory still doesn't exist. Also, I think it's superfluous to predict the entire name as there is only one .nc in the output folder. But I'm looking into it

Reply:
How about something like this:
```
outfile = validation_run.output_file
if outfile:
    table = get_img_stats(outfile.path)
    table = table.drop(columns = 'Group')
    return table
else:
    return None
```

Reply:
This would basically be your initial solution (which was already fine), but with a check whether the output_file attribute has already been set properly.

Also, I would say it's better to explicitly return `None` in case it doesn't exist, makes it easier to understand that this is intended behaviour.